_board/feather_huzzah.md

```diff
@@ -8,9 +8,33 @@ board_url: ""
 board_image: "/assets/images/boards/feather_huzzah.jpg"
 ---
 
-This board hasn't been fully documented yet. Please make a pull request adding more info to this file.
+Feather the open ecosystem format from Adafruit, and like its namesake it is thin, light, and lets you fly! Adafruit designed Feather to be an open standard for portable microcontroller cores.
```

Comment:
I think there's a verb missing in the first sentence?

Reply:
updated!